### DISCUSSION TITLE
I have a Mac Mini to donate to the project

### DISCUSSION THREAD

**Discussion by birnbacs** [helloSystem/hello#283] [General]
Hey probonopd & al, thanks for bringing this project along! 

I tried it and I love it, and for the same reasons you are doing it. Unfortunately it is not yet a replacement for productive work, although all we need here is a well-running LibreOffice, a browser and a few tools. Please keep up your work!

I read that you seek Mac hardware for development, so I seized the opportunity and purchased a Mac Mini at an auction recently. The Mac's probably ~10 years old and might be just what you asked for in the docs.

Drop me a message if you are interested. From the effort you put into setting up the keyboard properly you seem to be located in Europe, which means shipping won't cost an arm and a leg.

Kind regards

PS: sorry for opening a discussion on this, I could not find a better way of getting through to you

**Comment by probonopd** [comment]
Hello @birnbacs, thanks for contacting me and for your generosity. 

> Unfortunately it is not yet a replacement for productive work, although all we need here is a well-running LibreOffice, a browser and a few tools.

I'd be interested to hear what especially is preventing you from seeing it as a worthwhile replacement.

Regarding the Mac, please drop me a line at the [e-mail address given in the license file](https://github.com/helloSystem/ISO/blob/3462e5258fefdfc2dd6ae9aa6e9c1b0de0c54cdb/LICENSE#L4).

Thank you very much for supporting helloSystem!
